src/Angor/Client/Pages/Browse.razor

```diff
@@ -11,6 +11,7 @@
 @inject NavigationManager NavigationManager
 @inject IRelayService _RelayService
 @inject IIndexerService _IndexerService
+@attribute [StreamRendering(true)]
```

Comment:
Can this be explain more? I don't immediately see how it helps with fetching from indexer.
@DavidGershony thoughts?

Reply:
In Blazor, the `@attribute [StreamRendering(true)]` is used to enable streaming rendering for a specific Razor component. This attribute is particularly useful for optimizing the rendering process of Blazor components, especially when dealing with large amounts of data or complex UI structures.

Streaming rendering allows the server to send UI updates in smaller chunks rather than waiting for the entire component to render completely. This means that users can see parts of the UI rendered and become interactive sooner, while the rest of the UI continues to load and render in the background. This can significantly improve the perceived performance and responsiveness of your application.

Here's how it works:

- **Server-Side Blazor:** In server-side Blazor applications, this attribute allows the component to be rendered in a streaming fashion from the server to the client over a SignalR connection. This helps in reducing the initial load time and improving the interactivity of the application.
  
- **WebAssembly Blazor:** While this attribute is more relevant to server-side Blazor, in client-side scenarios (WebAssembly), the concept doesn’t apply as strictly, since the entire rendering process occurs in the browser. However, it can still be beneficial in scenarios where components are fetched dynamically.

Using this attribute can be especially beneficial in scenarios where you have components that take a significant amount of time to render due to complex calculations or data processing. By enabling streaming rendering, the user experience becomes smoother as parts of the UI are displayed as soon as they are ready.